template/functions.go

```diff
@@ -223,7 +226,16 @@ func formatFileSize(b int64) string {
 }
 
 func timeToRead(content string) int {
-	nbOfWords := len(strings.Fields(content))
+	sanitizedContent := strip.StripTags(content)
```

Comment:
Miniflux already have a function to strip tags. There is no need to add a new dependency.

https://github.com/miniflux/miniflux/blob/master/reader/sanitizer/strip_tags.go#L15

Reply:
Oh I didn't see that, thanks for the heads up! I'll be modifying it today.